views/templates/hook/admin/admin-order.tpl

```diff
@@ -66,6 +67,15 @@
                                         {/if}
                                     </a>
                                 </td>
+                                {if !$isAutomated}
+                                <td>
+                                    <form id="order-return-form" action="{$href}" method="post">
+                                        <button class="btn btn-link pt-0 pb-0" type="submit">
+                                            {l s='Download return label' mod='dpdbaltics'}
```

Comment:
Please format return label view button in a same way as it is with shipment buttpn